e2e/jest.config.js

```diff
@@ -18,9 +18,9 @@ module.exports = {
   preset: 'jest-puppeteer',
   globals: {
     SERVER_URL: 'http://localhost:3001',
-    JEST_TIMEOUT: 10000
+    JEST_TIMEOUT: 100000
```

Comment:
Is it because the 4th test takes so long?

Reply:
not really, it can remain as it was before. I rolled back the change.